### DISCUSSION TITLE
Does Alert history still work with prometheus?

### DISCUSSION THREAD

**Discussion by wuarthur** [prymitive/karma#5820] [General]
I found some conflicting information

[#5083](https://github.com/prymitive/karma/issues/5083) says all alerts is pulled from alert manager.

But in [#3684](https://github.com/prymitive/karma/issues/3684) it says the alert history is from Prometheus as described in the readme
![image](https://github.com/prymitive/karma/assets/15177124/b8d3aeee-8ab2-4701-b567-ab81086745ba)

**Comment by prymitive** [comment]
Which bit is unclear in the readme?
The list of currently firing alerts comes from alertmanager.
Prometheus provides information on how often each of those alerts found in alertmanager fired in the last 24h.
